ouroboros-network-framework/src/Ouroboros/Network/Server2.hs

```diff
@@ -90,6 +91,16 @@ data ServerArguments (muxMode  :: MuxMode) socket peerAddr versionData versionNu
       serverControlChannel        :: ServerControlChannel muxMode peerAddr versionData
                                                           bytes m a b,
 
+      -- | Governor control channel var is passed as an argument; this allows the Server
+      -- to communicate with the Outbound Governor telling it about new inbound
+      -- connections to add to the Known Peers Set.
+      --
+      governorControlChannel        :: GovernorControlChannel peerAddr m,
```

Comment:
I wouldn't do that by the server.  The connection manager is passing outbound duplex connections to the inbound governor, so why not let it pass inbound duplex connections to the outbound governor.  This will be more symmetric.

Reply:
See comment above. I'll update this comment though